CONTRIBUTING.md

```diff
@@ -310,42 +310,97 @@ We recommend you use IntelliJ as your IDE. The code style template for the proje
-* ### Commit Message Style
-    * **Separate subject from body with a blank line**
-    * **Subject**
-        * Limit the subject line to 10 words or 50 characters
-        * If you cannot make the subject short, you may be committing too many changes at once
-        * Capitalize the subject line
-        * Do not end the subject line with a period
-        * Use the imperative mood in the subject line
-    * **Body**
-        * Wrap the body at 72 characters
-        * Use the body to explain what and why versus how
-        * Use the indicative mood in the body\
-          For example, “If applied, this commit will ___________”
-        * Communicate only context (why) for the commit in the subject line
-        * Use the body for What and Why
-          * If your commit is complex or dense, share some of the How context
-        * Assume someone may need to revert your change during an emergency
-    * **Content**
-        * **Aim for smaller commits for easier review and simpler code maintenance**
-        * All bug fixes and new features must have associated tests
-        * Commits should pass tests on their own, not be dependent on other commits
-        * The following is recommended:
-            * Describe why a change is being made.
-            * How does it address the issue?
-            * What effects does the patch have?
-            * Do not assume the reviewer understands what the original problem was.
-            * Do not assume the code is self-evident or self-documenting.
-            * Read the commit message to see if it hints at improved code structure.
-            * The first commit line is the most important.
-            * Describe any limitations of the current code.
-            * Do not include patch set-specific comments.
-
-Details for each point and good commit message examples can be found on https://wiki.openstack.org/wiki/GitCommitMessages#Information_in_commit_messages
+
+### Conventional Commits
+We follow the [Conventional Commits](https://www.conventionalcommits.org/) specification for our commit messages and PR titles.
+
+**PR Title Format:**
+```
+<type>[(scope)]: <description>
+```
+
+**Types:**
+* **feat**: New feature or functionality
+* **fix**: Bug fix
+* **docs**: Documentation only changes
+* **style**: Code style changes (formatting, semicolons, etc)
+* **refactor**: Code refactoring without changing functionality
+* **perf**: Performance improvements
+* **test**: Adding or modifying tests
+* **build**: Build system or dependency changes
+* **ci**: CI/CD configuration changes
+* **chore**: Maintenance tasks
+* **revert**: Reverting a previous commit
+
+Note: Each PR/commit should have a single primary type. If your changes span multiple categories, choose the most significant one or consider splitting into separate PRs.
+
+**Scope (optional):** The area of code affected. Common scopes include:
+* `parser` - SQL parser and grammar
```

Comment:
Should we have a type for native engine ? We already use [native] prefix for PR titles.

Reply:
Done